tools/nni_cmd/launcher.py

```diff
@@ -120,9 +122,15 @@ def start_rest_server(port, platform, mode, config_file_name, experiment_id=None
         stderr_file.write(log_header)
         if sys.platform == 'win32':
             from subprocess import CREATE_NEW_PROCESS_GROUP
-            process = Popen(cmds, cwd=entry_dir, stdout=stdout_file, stderr=stderr_file, creationflags=CREATE_NEW_PROCESS_GROUP)
+            if args.foreground:
+                process = Popen(cmds, cwd=entry_dir, stdout=PIPE, stderr=STDOUT, creationflags=CREATE_NEW_PROCESS_GROUP)
```

Comment:
a little strange here, for win32, stderr is STDOUT, for linux, stderr is PIPE?

Reply:
https://stackoverflow.com/questions/10406532/python-subprocess-output-on-windows  
If I set `stderr=PIPE` in windows, the process will crash, so I redirect `stderr=STDOUT`.